src/lib/webengine/webview.cpp

```diff
@@ -154,6 +159,15 @@ void WebView::setPage(WebPage *page)
     // Set default zoom level
     zoomReset();
 
+    // Workaround < Qt5.9: apply zoom to every page loading progress
+    if (QVersionNumber::fromString(qVersion()) < QVersionNumber(5, 9, 0)) {
```

Comment:
I'm not prefer preprocessor directive cause Qupzilla can be compiled against different Qt version than runtime, qVersion() should return runtime version. I'll do it in static variable.